classroom.rb

```diff
@@ -0,0 +1,8 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = [] # Initialize an empty array to store the students associated with the classroom
+  end
+end
```

Comment:
- [ ] The classroom should have a method for the relationship with the student class. Kindly add it.

  ```
  def add_student(student)
    @students << student
    student.classroom = self
  end
  ```